Makefile

```diff
@@ -169,22 +174,22 @@ hosts-brave:
 hosts-brave-user:
 	@case $(OS) in \
 	Linux|*BSD) mkdir -p "${HOME}/.config/BraveSoftware/Brave-Browser/NativeMessagingHosts/"; \
-	            ln -sf "/usr/lib/browserpass/hosts/chromium/$(APP_ID).json" "${HOME}/.config/BraveSoftware/Brave-Browser/NativeMessagingHosts/" \
+	            ln -sf "$(LIB_DIR)/browserpass/hosts/chromium/$(APP_ID).json" "${HOME}/.config/BraveSoftware/Brave-Browser/NativeMessagingHosts/" \
 	            ;; \
 	Darwin)     mkdir -p "${HOME}/Library/Application Support/BraveSoftware/Brave-Browser/NativeMessagingHosts/"; \
-	            ln -sf "/usr/lib/browserpass/hosts/chromium/$(APP_ID).json" "${HOME}/Library/Application Support/BraveSoftware/Brave-Browser/NativeMessagingHosts/" \
+	            ln -sf "$(LIB_DIR)/browserpass/hosts/chromium/$(APP_ID).json" "${HOME}/Library/Application Support/BraveSoftware/Brave-Browser/NativeMessagingHosts/" \
 	            ;; \
 	*)          echo "The operating system $(OS) is not supported"; exit 1 ;; \
 	esac
 
 .PHONY: hosts-firefox
 hosts-firefox:
 	@case $(OS) in \
-	Linux)      mkdir -p "/usr/lib/mozilla/native-messaging-hosts/"; \
-	            ln -sf "/usr/lib/browserpass/hosts/firefox/$(APP_ID).json" "/usr/lib/mozilla/native-messaging-hosts/" \
+	Linux)      mkdir -p "$(LIB_DIR)/mozilla/native-messaging-hosts/"; \
+	            ln -sf "$(LIB_DIR)/browserpass/hosts/firefox/$(APP_ID).json" "/usr/lib/mozilla/native-messaging-hosts/" \
```

Comment:
@Infinisil what about this, this needs to use `$(LIB_DIR)` as well then, right?

Any other comments about the PR, or should I merge?

Reply:
Oh, didn't see this one. Using `LIB_DIR` (for the install directory) seems wrong, because afaik /usr/lib is hardcoded in firefox. On Nix/NixOS we patch Firefox build to use a custom native-messaging-hosts directory, and we install stuff ourselves there, so this makefile rule doesn't concern me. So yeah using /usr/lib makes more sense here I think.

Reply:
Alright, thanks 👍 

Final ask, could you please review the [Install on Nix / NixOS](https://github.com/browserpass/browserpass-native#install-on-nix--nixos) section, is it still correct?

Reply:
The NixOS part should still be correct. For the Nix/stateful part though, the current version of browserpass in nixpkgs doesn't have that Makefile included, so this needs an update (which I can do), and then it takes a while for it to be propagated to all release channels (assuming we backport the update).

It probably makes sense then to embed binaries in our browserpass package that call `make` on the Makefile with the appropriate `DESTDIR` and `PREFIX` set. Or maybe we could patch the makefile to have a different default for `DESTDIR`/`PREFIX`. I'll figure something out.

Anyways for now the second part of the instructions doesn't work, the previous version should still work though. I'll inform you when I got around to updating stuff for Nix.

Reply:
Alright, thanks!